db/migrate/20221123134326_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :authorId
+      t.integer :postId
```

Comment:
- [ ] These should be `:author_id` and `:post_id` respectively. Kindly fix that.

- [ ] These changes should be reflect in the schema file.